### PR TITLE
Fix: certificate display US country only

### PR DESCRIPTION
**Pull request by mactanxin**
ticket: #19502

### CODE REVIEW

File: ui/sections/system/inspectors/crypto-certificate.reel/crypto-certificate-creation.reel/crypto-certificate-creation.js [hunk 1]
```diff
@@ -48,7 +48,8 @@ exports.CryptoCertificateCreation = AbstractInspector.specialize(/** @lends Cryp
             if (this.object && this.object.type === CryptoCertificateType.CA_INTERNAL) {
                 this.object.selfsigned = true;
             }
-            if (this.object) {
+            if (this.object._isNew == true) {
```
Comment (pchaussalet) [inline ui/sections/system/inspectors/crypto-certificate.reel/crypto-certificate-creation.reel/crypto-certificate-creation.js]: this.object._isNew is a boolean, no need to compare it to `true`

**Review by pchaussalet — CHANGES_REQUESTED** [review]

**Comment by pchaussalet** [comment]
there is a conflict on `ui/inspectors/system-section.reel/mail.reel/mail.html`

**Comment by mactanxin** [comment]
updated. conflict fixed

**Review by pchaussalet — APPROVED** [review]
LGTM

**Pull request merged**